src/peep/staff.c

```diff
@@ -78,6 +78,40 @@ void game_command_update_staff_colour(int *eax, int *ebx, int *ecx, int *edx, in
 	*ebx = 0;
 }
 
+
+/*
+Select random skins for entertainers on palcement.
++ only select valid skins for the current scenery
+*/
+int staff_select_random_skin(int staff_type)
+{
+	if (staff_type != 3)
+		return 0;
+
+	int ebx = 0;
+	staff_get_valid_skins(&ebx);
+
+	uint16 no_entries = 0;
```

Comment:
no_entries is a relict out of the window/staff.c; but I'll edit it, no problem :D